app/src/main/java/sims/michael/gerritslackbot/slack/EventGroupToSlackMessageTransformer.kt

```diff
@@ -53,7 +53,7 @@ class EventGroupToSlackMessageTransformer(
                 channel = slackName,
                 attachments = listOf(Attachment(
                         fallback = "Summary of changes",
-                        text = events.joinToString("\n") { event -> event.toSlackSummary() }
+                        text = events.map { it.toSlackSummary() }.joinToString("\n")
```

Comment:
Why this change? The previous form is simpler (and IDE inspections suggests it)

Reply:
Not sure now but a I thought that a have seen this form in another place of this file. So there were there were two places doing same thing but with different syntax/construction so I just picked one made this parts of code use same construct.